src/utils/generateMonitoringData.js

```diff
@@ -36,83 +36,103 @@ export const generateMonitoringStats = (data, navigate, dispatch, tab) => {
   const navigateToJobsMonitoringPage = (modalFilters, filters = {}, dateFutureOption) => {
     const datePickerOptions = dateFutureOption ? datePickerFutureOptions : datePickerPastOptions
     let date = datePickerOptions.find(
-      option => option.id === (dateFutureOption ? NEXT_24_HOUR_DATE_OPTION : PAST_24_HOUR_DATE_OPTION)
+      option =>
+        option.id === (dateFutureOption ? NEXT_24_HOUR_DATE_OPTION : PAST_24_HOUR_DATE_OPTION)
     )
 
-    dispatch(setFilters(
-      {
+    dispatch(
+      setFilters({
         ...filters,
         saveFilters: true,
         dates: {
           value: date.handler(dateFutureOption),
           isPredefined: date.isPredefined,
           initialSelectedOptionId: date.id
         }
-      }
-    ))
+      })
+    )
     dispatch(setModalFiltersValues({ name: tab, value: modalFilters }))
     navigate(`/projects/jobs-monitoring/${tab}`)
   }
 
   return tab === JOBS_MONITORING_JOBS_TAB
     ? {
-      all: {
-        counter: data.all,
-        link: () => navigateToJobsMonitoringPage({ state: ['all'] })
-      },
-      counters: [
-        {
-          counter: data.running,
-          link: () =>
-            navigateToJobsMonitoringPage({
-              state: ['running', 'created', 'pending', 'unknown', 'aborting']
-            }),
-          statusClass: 'running'
-        },
-        {
-          counter: data.failed,
-          link: () => navigateToJobsMonitoringPage({ state: ['error', 'aborted'] }),
-          statusClass: 'failed'
-        },
-        {
-          counter: data.completed,
-          link: () => navigateToJobsMonitoringPage({ state: ['completed'] }),
-          statusClass: 'completed'
-        }
-      ]
-    }
-    : tab === JOBS_MONITORING_WORKFLOWS_TAB
-      ? {
         all: {
           counter: data.all,
-          link: () => navigateToJobsMonitoringPage({ state: ['all'] }, { groupBy: GROUP_BY_WORKFLOW })
+          link: () => navigateToJobsMonitoringPage({ state: ['all'] })
         },
         counters: [
           {
             counter: data.running,
-            link: () => navigateToJobsMonitoringPage({ state: ['running'] }, { groupBy: GROUP_BY_WORKFLOW }),
-            statusClass: 'running'
+            link: () =>
+              navigateToJobsMonitoringPage({
+                state: ['running', 'created', 'pending', 'unknown', 'aborting']
+              }),
+            statusClass: 'running',
+            tooltip: ['Aborting', 'Pending', 'Running']
```

Comment:
I don't see any purpose of using tooltip as an array. Better to use just a string so that we wouldn't need to join its items.